charts/gsp-cluster/values.yaml

```diff
@@ -897,3 +897,5 @@ gatekeeper:
 
 cert-manager:
   replicaCount: 2
+  securityContext:
```

Comment:
Need to set `enabled: true` for this to be used, and also set `runAsUser`

Reply:
(see https://github.com/jetstack/cert-manager/blob/release-0.11/deploy/charts/cert-manager/templates/deployment.yaml#L49)

Reply:
no to both. `securityContext.enabled` is deprecated according to the chart docs: https://github.com/jetstack/cert-manager/tree/master/deploy/charts/cert-manager

The preferred method is to pass in a raw PodSecurityContext block and it just whacks it into the Deployment

Reply:
here's the template itself: https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/templates/deployment.yaml#L59-L67

Reply:
That's the docs for master.

Reply:
doh 🤦‍♂ pushed a fix.  We don't need to set `runAsUser` though because it has a defaulted value in the chart's values.yaml.